src/cli/actions/defaultAction.ts

```diff
@@ -104,6 +104,9 @@ const buildCliConfig = (options: CliOptions): RepomixConfigCli => {
   if (options.outputShowLineNumbers !== undefined) {
     cliConfig.output = { ...cliConfig.output, showLineNumbers: options.outputShowLineNumbers };
   }
+  if (options.copyToClipboard) {
+    cliConfig.output = { ...cliConfig.output, copyToClipboard: true };
```

Comment:
```suggestion
    cliConfig.output = { ...cliConfig.output, copyToClipboard: options.copyToClipboard };
```

Reply:
Sorry, this was my mistake in the suggested code.

The value should come from the options rather than being hardcoded to true.